.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Indentation in this file should be set to 2 spaces throughout as it is in the rest of the file